docs/reference/github/workflow/issues.md

```diff
@@ -36,8 +36,7 @@ At this point, you should give a succinct title and describe the issue in the te
 sections to fill out. Try to give only the necessary details to make a clear and concise issue. If you are unsure on how
 to construct your issue, take a look at current or past issues and ask the software leads for further guidance if necessary.
 
-Finally, feel free to [make suggestions](https://github.com/UBCSailbot/.github/issues/templates/edit) on new templates or
```

Comment:
Instead of removing this link because it doesn't pass the link check, we can ignore it in `.markdown-link-check.json`

Reply:
For example

```json
  "ignorePatterns": [
    {
      "pattern": "^http://example.net"
    }
  ],
```